src/stories/Table.js

```diff
@@ -48,6 +49,36 @@ const columns = [
   }
 ]
 
+class HOC extends React.Component {
+  constructor () {
+    super()
+
+    this.state = {
+      PokeDex: _.cloneDeep(PokeDex)
+    }
+
+    this.changeName = this.changeName.bind(this)
+  }
+
+  changeName (e) {
+    const { PokeDex } = this.state
+
+    PokeDex[0] = Object.assign({}, PokeDex[0], { Name: e.target.value })
```

Comment:
I would like to also see a `slice()` here since it's a best practice, even though the table component will render on each `setState()`.